README.md

```diff
@@ -224,7 +224,7 @@ This repository uses a very simple initialisation script, [./setup_repo.sh](./se
 <<__role_arn.cluster_autoscaler__>>=arn:aws:iam::123456789012:role/my-cluster_kube-system_aws-cluster-autoscaler
 ```
 
-The init script will look for all occurences in the ./distribution folder of the placeholder `<<__role_arn.cluster_autoscaler__>>` and will replace it with the value `arn:aws:iam::123456789012:role/my-cluster_kube-system_aws-cluster-autoscaler`. Please note that that comments (`//`, `#`), quatation marks (`"`, `'`) or unnecessary line-breaks should be avoided.
+The init script will look for all occurences in the ./distribution folder of the placeholder `<<__role_arn.cluster_autoscaler__>>` and will replace it with the value `arn:aws:iam::123456789012:role/my-cluster_kube-system_aws-cluster-autoscaler`. Please note that that comments (`//`, `#`), quotation marks (`"`, `'`) or unnecessary line-breaks should be avoided.
```

Comment:
Good catch. Looks like there's another one here.
```suggestion
The init script will look for all occurences in the ./distribution folder of the placeholder `<<__role_arn.cluster_autoscaler__>>` and will replace it with the value `arn:aws:iam::123456789012:role/my-cluster_kube-system_aws-cluster-autoscaler`. Please note that comments (`//`, `#`), quotation marks (`"`, `'`) or unnecessary line-breaks should be avoided.
```

Reply:
there are a few more actually I'll be updating this PR